src/pages/Search/Search.js

```diff
@@ -140,6 +163,53 @@ export default class Search extends Component {
           });
       }
   }
+  handleDateOptionDisplay = (selectedDateValue) => {
+    console.log("YOLO");
+    console.log('selectDateObject ' + selectedDateValue);
+    console.log(typeof selectedDateValue);
+    const {from, to} = this.state;
+    const modifiers = {start: from, end: to};
+    if (selectedDateValue === undefined) return;
+    console.log('dateOption Choice ' + selectedDateValue);
+    //trigger={<button> Trigger</button>}
+    if (selectedDateValue === 'dayOfWeek') {
+        return (
+            <>
+              <div className="headingRow"> Day of the Week </div>
```

Comment:
I think the "Day of the week" div should be inside of `<Popup>` then you can remove the `<>`
<img width="354" alt="Screenshot 2019-07-15 14 38 30" src="https://user-images.githubusercontent.com/10586811/61240018-3c074180-a70e-11e9-8516-7653cf23ff55.png">